### PR TITLE
Third Commit

### PR DESCRIPTION
**Pull request by Amitgupta76**

### CODE REVIEW

**Comment by pvishnuprasaath** [comment]
congo for your first PR! 🎉

File: src/Components/InputForm.test.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import InputForm from './InputForm';
+
+test('InputForm renders correctly', () => {
```
Comment (pvishnuprasaath) [inline src/Components/InputForm.test.tsx]: add a describe block to wrap all the tests inside. Look into the CTP test case files

**Review by pvishnuprasaath — CHANGES_REQUESTED** [review]

**Comment by pvishnuprasaath** [comment]
Update the PR title and description as discussed

File: src/Components/InputForm.test.tsx (hunk 1 above)
Comment (Amitgupta76) [reply src/Components/InputForm.test.tsx]: Fine I'll make the required changes.

**Comment by Amitgupta76** [comment]
> Update the PR title and description as discussed

I'll do that as well.

**Pull request closed without merging**